AUTHORS

```diff
@@ -1,21 +1,631 @@
+Beat Bolli <dev@drbeat.li>
+Ben Laurie <ben@links.org>
+Ben Laurie <benl@benl-macbookpro3.local>
+Ben Laurie <ben@openssl.org>
+<ben@algroup.co.uk>
+<ben@cryptix.org>
+<ben@thebunker.net>
+Benjamin Kaduk <bkaduk@akamai.com>
+Bernd Edlinger <bernd.edlinger@hotmail.de>
+<bet@rahul.net>
+<bg@sics.se>
+Bill Cox <waywardgeek@google.com>
+Billy Brumley <bbrumley@gmail.com>
+<bixby@hp.com>
+Bjoern Zeeb <bzeeb@zabbadoz.net>
+<blong@fiction.net>
+<bm@ulf.mali.sub.org>
+<bmoeller@acm.org>
+<bmoeller@openssl.org>
+Bodo Moeller <bodo@openssl.org>
+Bodo Moeller <bmoeller@google.com>
+<bowe@chip.ma.certco.com>
+<bowman@math.utah.edu>
+<brad@mainstreetsoftworks.com>
+<brendan.kennedy@intel.com>
+<brian.carpenter@gmail.com>
+<bruce.stephens@isode.com>
+<bugs@openssl.org>
+<carenas@jmconsultores.com.pe>
+Carl Jackson <carl@avtok.com>
+Carlos Alberto Lopez Perez <clopez@igalia.com>
+<carson@taltos.org>
+<cato@df.lth.se>
+Cesar Pereida <cesar.pereida@aalto.fi>
+<cfe-commits@cs.uiuc.edu>
+<chaffee@plateau.cs.berkeley.edu>
+<china@thewrittenword.com>
+<chk@pobox.com>
+Chris Rorvick <chris@rorvick.com>
+Chris Watts <cwatts1@us.ibm.com>
+<chris@qwirx.com>
+Christian Bundy <christianbundy@fraction.io>
+Christian Heimes <cheimes@redhat.com>
+Christian Heimes <christian@python.org>
+clucey <catriona.lucey@intel.com>
+Conrado Porto Lopes Gouva <conradoplg@gmail.com>
+Corinna Vinschen <vinschen@redhat.com>
+Cory Benfield <lukasaoz@gmail.com>
+Coty Sutherland <csutherl@redhat.com>
+<crispin@flowerday.cx>
+Cristian Rodrguez <crrodriguez@opensuse.org>
+Cristian Stoica <cristian.stoica@nxp.com>
+Cynh <cynh@hotmail.fr>
+<dalgoda@ix.netcom.com>
+<dan@bti.net>
+Daniel Black <daniel.black@openquery.com>
+Daniel Kahn Gillmor <dkg@fifthhorseman.net>
+<daniel.m@sent.com>
+<daniel.subs@internode.on.net>
+<danielml@sent.com>
+Dario B <darizotas@gmail.com>
+Darren Tucker <dtucker@google.com>
+<davem@davemloft.net>
+<davem@devemloft.net>
+David Bar <david.bar@gmail.com>
+David Benjamin <davidben@google.com>
+David Benjamin <davidben@chromium.org>
+David Lloyd <david.lloyd@fsmail.net>
+David Ramos <daramos@stanford.edu>
+David Woodhouse <dwmw2@infradead.org>
+David Woodhouse <David.Woodhouse@intel.com>
+Davide Galassi <davide.galassi@gmail.com>
+<dean@arctic.org>
+<deengert@anl.gov>
+<desnacked@gmail.com>
+<dg@sunet.ru>
+Dirk Feytons <dirk.feytons@gmail.com>
+Dirk Feytons <dirk.feytons@technicolor.com>
+<dirk.meyer@dinoex.sub.org>
+<djm@mindrot.org>
+DK <dmitrykos@neutroncode.com>
+<dkaufman@rahul.net>
+<dmiller@ilogic.com.au>
+Dmitry Belyavskiy <beldmit@gmail.com>
+Dmitry Sobinov <sobinoff@gmail.com>
+Doug Goldstein <cardoe@gentoo.org>
+Doug Hogan <doug@acyclic.org>
+<Doug.Erickson@Part.NET>
+Douglas E Engert <deengert@gmail.com>
+<douglas.stebila@sun.com>
+<douglas@stebila.ca>
+<dougopenssl@claar.org>
+Dr Stephen Henson <shenson@drh-consultancy.co.uk>
+Dr. Lutz Janicke <lutz@lutz-jaenicke.de>
+Dr. Matthias St. Pierre <Matthias.St.Pierre@ncp-e.com>
+Dr. Stephen Henson <steve@openssl.org>
```

Comment:
Two Dr. Stephen Hensons ?

Reply:
Three, actually.  `<drh@celocom.com>` (below this comment) is him as well, and if you grep for `shenson@`, you'll find yet a few more a bunch of lines down.

Reply:
There could be a zillion; they all should stay for reasons I described.